sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -52,3 +52,16 @@ export interface FileCreateDto {
   size: number;
   mimetype: string;
 }
+
+export class StudentEducationProgramDto {
```

Comment:
Is this DTO used to send data to the API?
Using a class and the validators have effect only to submit data to the API. If this is not the case, please convert it to an interface and remove the validators.